data/json/overmap_terrain.json

```diff
@@ -6886,6 +6886,36 @@
         "see_cost" : 5,
         "mondensity" : 2,
         "sidewalk" : true
+    },{
+        "type" : "overmap_terrain",
+        "id" : "roadstop",
+        "name" : "roadstop",
+        "rotate" : true,
+        "sym" : 94,
+        "color" : "light_blue",
+        "see_cost" : 0,
+        "mondensity" : 2,
+        "sidewalk" : true
+    },{
+        "type" : "overmap_terrain",
+        "id" : "roadstop_a",
+        "name" : "public washroom",
+        "rotate" : true,
+        "sym" : 94,
+        "color" : "light_blue",
+        "see_cost" : 0,
+        "mondensity" : 2,
+        "sidewalk" : true
+    },{
+        "type" : "overmap_terrain",
+        "id" : "roadstop_b",
+        "name" : "roadside foodcart",
+        "rotate" : true,
+        "sym" : 94,
+        "color" : "orange",
```

Comment:
Ok, I see it now. I was focusing on the top entry. You're right, orange is not in the pallet. I've replaced the entry with magenta.